sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/registries.json

```diff
@@ -516,6 +516,11 @@
           },
           "x-ms-identifiers": [],
           "x-nullable": true
+        },
+        "managedResourceGroupTags": {
+          "description": "Tags to be applied to the managed resource group associated with this registry.",
```

Comment:
Please update the task that I shared with you internally so we keep track of this changes in the swagger

Reply:
I'm auditing all my task to address this, closing this issue to avoid confusion about whether this is ready to merge.